qiita_ware/test/test_ebi.py

```diff
@@ -239,10 +239,31 @@ def test_generate_curl_command(self):
                '-F "SAMPLE=@%(xml_dir)s/sample.xml" '
                '-F "RUN=@%(xml_dir)s/run.xml" '
                '-F "EXPERIMENT=@%(xml_dir)s/experiment.xml" '
-               '"ebi_dropbox_url/?auth=ERA%%20ebi_seq_xfer_user'
-               '%%20ebi_access_key%%3D"') % {'xml_dir': submission.xml_dir}
+               '"ebi_dropbox_url/?auth=ENA%%20ebi_seq_xfer_user'
+               '%%20ebi_access_key"') % {'xml_dir': submission.xml_dir}
         self.assertEqual(obs, exp)
 
+    def test_parse_EBI_reply(self):
```

Comment:
Just to make it clear. This function is going to be refactored once we have the DB changes to parse all the EBI values, correct?